Sources/Core/Client/Client.swift

```diff
@@ -121,7 +121,7 @@ public final class Client {
         }
     }
     
-    func reset() {
+    public func reset() {
```

Comment:
Looks good.
Can you add a doc for the method and point developers to use `Client.set(user:, token:)` to restore the connection.

Reply:
And now better to rename it to `disconnect()`.

Reply:
Agreed and good idea. Will update.